02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,71 @@
+const { people } = require('../data')
+
+
+const getPeople = (req, res) => {
+    res.status(200).json({ success: true, data: people });
+}
+
+
+const createPerson = (req, res) => {
+    const { name } = req.body
+    if (!name) {
+        return res.status(400)
+            .json({ success: false, msg: "Please provide a name" })
+    }
+    // adds the person to the json list
+    people.push({
+        id: people.length + 1, name: name
+    })
+
+    res.status(201).send({ success: true, person: name })
+}
+
+
+const createPersonPostman = (req, res) => {
```

Comment:
Ah to clarify the assignment, it meant you should test your existing create and get methods using a postman call, not create a separate endpoint for them